model/pdata/generated_metrics.go

```diff
@@ -1438,6 +1438,50 @@ func (ms HistogramDataPoint) SetFlags(v MetricDataPointFlags) {
 	(*ms.orig).Flags = uint32(v)
 }
 
+// Min_Type returns the type of the min_ for this HistogramDataPoint.
+// Calling this function on zero-initialized HistogramDataPoint will cause a panic.
+func (ms HistogramDataPoint) Min_Type() OptionalType {
+	switch ms.orig.Min_.(type) {
+	case *otlpmetrics.HistogramDataPoint_Min:
+		return OptionalTypeDouble
+	}
+	return OptionalTypeNone
+}
+
+// Min returns the min associated with this HistogramDataPoint.
+func (ms HistogramDataPoint) Min() float64 {
+	return (*ms.orig).GetMin()
+}
+
+// SetMin replaces the min associated with this HistogramDataPoint.
+func (ms HistogramDataPoint) SetMin(v float64) {
+	(*ms.orig).Min_ = &otlpmetrics.HistogramDataPoint_Min{
```

Comment:
Nit: this can check to see if `(*ms.orig).Min_ != nil` and use it without allocating a new one.